packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -505,6 +525,12 @@ export default mixins(
 				this.versionControlStore.sync({ commitMessage: prompt.value });
 			}
 		},
+		onUpgradePlanClicked() {
+			location.href =
+				this.settingsStore.settings.license.environment === 'production'
+					? CHANGE_PLAN_PAGE_PRODUCTION
+					: CHANGE_PLAN_PAGE_STAGING;
+		},
```

Comment:
just tested this on staging.. unfortunately this is not getting set correctly right now for licenses... would require an env var change for staging to work.. 
<img width="1440" alt="Bildschirmfoto 2023-05-05 um 11 05 08" src="https://user-images.githubusercontent.com/4711238/236418662-cd2ab68e-e56a-4b3a-b673-5c946152089e.png">

Reply:
I guess we set the proper env in staging then

Reply:
Looks like we are using this already on the usage store 

https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/stores/usage.store.ts#L54

Reply:
this is different.. does not matter for cloud staging.. because we don't show this page at all there.. this mostly for development